backend/services/coworking/reservation.py

```diff
@@ -519,10 +550,9 @@ def _get_reservable_rooms(self) -> Sequence[RoomDetails]:
         Returns:
             Sequence[RoomDetails]: A sequence of RoomDetails models representing all the reservable rooms, excluding room 'SN156'.
         """
-
         rooms = (
             self._session.query(RoomEntity)
-            .where(RoomEntity.reservable == True)
+            .where(or_(RoomEntity.reservable == True, RoomEntity.id == 'SN156'))
```

Comment:
Let's think of ways of achieving the graying out for drop-in/desk reservations without this logic in the future.

Reply:
Agreed, this was something that felt uncomfortable to me as well! I have a couple of ideas already in mind that might work with graying out drop-in reservations and circumvent this logic that I will try out this week.